src/core/metrics/app-metrics.ts

```diff
@@ -59,34 +64,46 @@ export class AppMetrics {
   }
 
   private constructor() {
-    globalThis.KONTUR_METRICS = {
-      watchList: this.watchList,
-      markers: this.markers,
-      sequences: this.sequences,
-      events: this.eventLog,
-      toggleAlert: () => this.settings.toggle('KONTUR_SQ_ALERT'),
-      toggleLog: () => this.settings.toggle('KONTUR_SQ_LOG'),
-    };
+    //
   }
 
-  init(appId: string, route: string) {
-    console.info('appMetrics.init with', appId, route);
+  init(appId: string, route: string, hasFeature: (f: AppFeatureType) => boolean): void {
     // currently we support metrics only for map page
     if (route !== '') {
       // '' is route for map
       return;
     }
 
     this.reportTemplate.appId = appId ?? '';
+
+    // add available features to metrics
+    this.watchList = buildWatchList(hasFeature);
```

Comment:
so it's called `watchList` and not smth like `featuresList` because we not only watching features right?

Reply:
we are watching for events, but list of events depends on enabled features